diff --git a/node_modules/@graphql-codegen/testing/index.cjs.js b/node_modules/@graphql-codegen/testing/index.cjs.js
index 0df1c42..d6050a9 100644
--- a/node_modules/@graphql-codegen/testing/index.cjs.js
+++ b/node_modules/@graphql-codegen/testing/index.cjs.js
@@ -7,7 +7,7 @@ function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'defau
 const commonTags = require('common-tags');
 const path = require('path');
 const fs = require('fs');
-const diff = _interopDefault(require('jest-diff'));
+const {diff} = _interopDefault(require('jest-diff'));
 const typescript = require('typescript');
 const open = _interopDefault(require('open'));
 
diff --git a/node_modules/@graphql-codegen/testing/index.esm.js b/node_modules/@graphql-codegen/testing/index.esm.js
index 38c193c..c299a15 100644
--- a/node_modules/@graphql-codegen/testing/index.esm.js
+++ b/node_modules/@graphql-codegen/testing/index.esm.js
@@ -1,7 +1,7 @@
 import { oneLine, stripIndent } from 'common-tags';
 import { resolve, join, dirname } from 'path';
 import { existsSync } from 'fs';
-import diff from 'jest-diff';
+import {diff} from 'jest-diff';
 import { ModuleResolutionKind, ScriptTarget, JsxEmit, ModuleKind, createSourceFile, ScriptKind, flattenDiagnosticMessageText, createCompilerHost, createProgram } from 'typescript';
 import open from 'open';
 
